src/common/tiglcommonfunctions.h

```diff
@@ -300,6 +300,21 @@ TIGL_EXPORT opencascade::handle<Geom_BSplineCurve> ApproximateArcOfCircleToRatio
 TIGL_EXPORT TopoDS_Wire BuildWireRectangle(const double heightToWidthRatio, const double cornerRadius=0.0,
                                            const double tol=Precision::Approximation());
 
+/**
+ * @brief BuildWireSuperellipse Builds a superelliptic wire in (y,z) - plane with width 1 and height 1
+ * @param lowerHeightFraction Fraction of height of the lower semi_ellipse relative to the toal height
+ * @param mLower Exponent m for lower semi-ellipse
+ * @param mUpper Exponent m for upper semi-ellipse
+ * @param nLower Exponent n for lower semi-ellipse
+ * @param nUpper Exponent n for upper semi-ellipse
+ * @param tol
```

Comment:
Can we add an explanation as to why a tolerance is used? It is not clear from the documentation and syntax that the superellipse is being approximated by a BSpline and this tolerance controls how good the approximation is. I suppose the same can be said about the BuildWireRectangle function.